src/test/resources/test_data/overall_calculation_response/tranches/crs-2077.json

```diff
@@ -3,7 +3,7 @@
     "SLED": "2025-02-25",
     "CRD": "2024-09-10",
     "TUSED" : "2025-08-26",
-    "HDCED": "2024-07-12",
+    "HDCED": "2024-08-31",
```

Comment:
can someone double check this as well please as my reading of crs-2077 is that the HDCED should be 2024-08-31

https://dsdmoj.atlassian.net/browse/CRS-2077